docs/notebooks/deep_cde.ipynb

```diff
@@ -0,0 +1,643 @@
+       "<tr><td>DeepGP.f_layers[1].kernel.lengthscales     </td><td>Parameter</td><td>Softplus        </td><td>       </td><td>True       </td><td>(2,)       </td><td>float64</td><td>[0.05 0.2 ]                 </td></tr>\n",
+       "<tr><td>DeepGP.f_layers[1].inducing_variable.Z     </td><td>Parameter</td><td>Identity        </td><td>       </td><td>True       </td><td>(20, 2)    </td><td>float64</td><td>[[0.04166667, 0.66191201... </td></tr>\n",
+       "<tr><td>DeepGP.f_layers[1].q_mu                    </td><td>Parameter</td><td>Identity        </td><td>       </td><td>True       </td><td>(20, 1)    </td><td>float64</td><td>[[0....                     </td></tr>\n",
+       "<tr><td>DeepGP.f_layers[1].q_sqrt                  </td><td>Parameter</td><td>FillTriangular  </td><td>       </td><td>True       </td><td>(1, 20, 20)</td><td>float64</td><td>[[[1., 0., 0....            </td></tr>\n",
+       "<tr><td>DeepGP.f_layers[2].kernel.variance         </td><td>Parameter</td><td>Softplus        </td><td>       </td><td>True       </td><td>()         </td><td>float64</td><td>1.0                         </td></tr>\n",
+       "<tr><td>DeepGP.f_layers[2].kernel.lengthscales     </td><td>Parameter</td><td>Softplus        </td><td>       </td><td>True       </td><td>()         </td><td>float64</td><td>1.0                         </td></tr>\n",
+       "<tr><td>DeepGP.f_layers[2].inducing_variable.Z     </td><td>Parameter</td><td>Identity        </td><td>       </td><td>True       </td><td>(20, 1)    </td><td>float64</td><td>[[2.48779388...             </td></tr>\n",
+       "<tr><td>DeepGP.f_layers[2].q_mu                    </td><td>Parameter</td><td>Identity        </td><td>       </td><td>True       </td><td>(20, 1)    </td><td>float64</td><td>[[0....                     </td></tr>\n",
+       "<tr><td>DeepGP.f_layers[2].q_sqrt                  </td><td>Parameter</td><td>FillTriangular  </td><td>       </td><td>True       </td><td>(1, 20, 20)</td><td>float64</td><td>[[[1.e-05, 0.e+00, 0.e+00...</td></tr>\n",
+       "<tr><td>DeepGP.likelihood_layer.likelihood.variance</td><td>Parameter</td><td>Softplus + Shift</td><td>       </td><td>False      </td><td>()         </td><td>float64</td><td>0.009999999999999998        </td></tr>\n",
+       "</tbody>\n",
+       "</table>"
+      ]
+     },
+     "metadata": {}
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Fit\n",
+    "\n",
+    "We can now fit the model. Because of the `DirectlyParameterizedEncoder` it is important to set the batch size to the number of datapoints and turn off shuffle. This is so that we use the associated latent variable for each datapoint. If we would use an Amortized Encoder network this would not be necessary."
```

Comment:
"It is important to set the batch size to the number of datapoints and turn off shuffle, checkout `DirectlyParameterizedEncoder` for details"? This needs a bit of explanation, what `DirectlyParameterizedEncoder` is.